src/Harald.IntegrationTests/Facades/Slack/SlackFacadeTest.cs

```diff
@@ -5,18 +5,20 @@
 using System.Threading.Tasks;
 using Harald.WebApi.Infrastructure.Facades.Slack;
 using Harald.WebApi.Infrastructure.Serialization;
+using Xunit.Priority;
 
 namespace Harald.IntegrationTests.Facades.Slack
 {
+    [TestCaseOrderer(PriorityOrderer.Name, PriorityOrderer.Assembly)]
     public class SlackFacadeTest
     {
-        [Fact]
+        [Fact, Priority(0)]
         public async Task CreateChannel_Given_valid_input_Should_create_channel()
         {
             // Arrange
             var httpClient = GetHttpClient();
             var sut = new SlackFacade(httpClient, new JsonSerializer(), new SlackHelper());
-            var channelName = "janie-test";
+            var channelName = "ded-team-one";

```

Comment:
Maybe elevate this to constant in class?

Reply:
Went ahead and changed this line and other vars where applicable in the same file to a const. Can be seen here fe5fd7124403ecc900daad96c1be689e493860bd